### PR TITLE
Save scroll position for Session list

### PR DESCRIPTION
**Pull request by aagam-shah**

### CODE REVIEW

**Comment by maxme** [comment]
Edge case: that won't work if the list activity is killed - we should save/restore the state in onSaveInstanceState / onViewCreated

**Comment by aagam-shah** [comment]
@maxme made changes as mentioned

File: app/src/main/java/org/wordcamp/android/wcdetails/SessionsFragment.java [hunk 1]
```diff
@@ -60,28 +62,33 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
...
+                    Intent detail = new Intent(getActivity(), SessionDetailsActivity.class);
+                    detail.putExtra("session", sessionDBList.get(position));
+                    getActivity().startActivity(detail);
+                }
+            });
+
+            sessionList.getWrappedList().setHeaderDividersEnabled(true);
+            sessionList.setAdapter(sessionsListAdapter);
+        } else {
+            sessionList.onRestoreInstanceState(savedInstanceState.getParcelable("sessionListState"));
```
Comment (maxme) [inline app/src/main/java/org/wordcamp/android/wcdetails/SessionsFragment.java]: `sessionList` not initialized here

Comment (aagam-shah) [inline app/src/main/java/org/wordcamp/android/wcdetails/SessionsFragment.java]: Good catch, fixed it.

**Comment by maxme** [comment]
:shipit:

**Pull request merged**